src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -27,34 +34,47 @@ Tab::~Tab()
 void Tab::_MakeTabViewItem()
 {
     _tabViewItem = ::winrt::Microsoft::UI::Xaml::Controls::TabViewItem{};
-    const auto title = _control.Title();
-
-    _tabViewItem.Header(title);
+    _tabViewItem.FontSize(12);

```

Comment:
unrelated change, should we keep?

Reply:
So it's sorta related. I was trying to visually join the tabs to the pane splits. Turns out if you make the tab text just a bit smaller (the current value is 14), we'll get just a tiny amount of an underline under the tabs that it'll link to.
![image](https://user-images.githubusercontent.com/18356694/57874161-913dda00-77d5-11e9-905d-314a01fa6482.png)

If this is unwanted, I can change it for sure.